browser/net/brave_static_redirect_network_delegate_helper.cc

```diff
@@ -48,6 +48,8 @@ int OnBeforeURLRequest_StaticRedirectWorkForGURL(
       URLPattern::SCHEME_HTTP | URLPattern::SCHEME_HTTPS, kCRXDownloadPrefix);
   static URLPattern autofill_pattern(
       URLPattern::SCHEME_HTTPS, kAutofillPrefix);
+  static URLPattern crowdDeny_pattern(
```

Comment:
we should not continue to follow this `*_pattern` convention that is not consistent with the style guide.